python/source/data.rst

```diff
@@ -137,3 +137,57 @@ function
 .. testoutput::
 
   0 .. 198
+
+Searching for values matching a predicate in Arrays
+---------------------------------------------------
+
+If you have to look for values matching a predicate in Arrow arrays
+the :mod:`arrow.compute` module provides a bunch of predicates that
+can be used to find the values you are looking for.
```

Comment:
```suggestion
If you have to look for values matching a predicate in Arrow arrays
the :mod:`arrow.compute` module provides several methods that
can be used to find the values you are looking for.
```

"a bunch of" is a little informal.  Function might be a better word than method, I'm not sure.  I think we should distinguish between "predicate" (the condition we want satisfied) and "method/function" (the tool we use to figure out what satisfies that condition).  A method/function can be a predicate but I think it should return a bool (and not an array of bool) in that case.  For example `lambda x: x > 10` is a predicate but `filter(lambda x: x > 10, my_list)` is not.